File: clang/test/Driver/msvc-link.cpp
// REQUIRES: system-windows
//
// RUN: %clang -### %s -Ltest 2>&1 | FileCheck %s
//
// Test that user provided paths come before compiler-rt
// CHECK: "-libpath:test"
// CHECK: "-libpath:{{.*}}\\lib\\clang\\{{[0-9]+}}\\lib\\windows"

File: clang/test/Driver/flang/msvc-link.f90
! RUN: %clang --driver-mode=flang --target=x86_64-pc-windows-msvc -### %s -Ltest 2>&1 | FileCheck %s
!
! Test that user provided paths come before the Flang runtimes
! CHECK: "-libpath:test"
! CHECK: "-libpath:{{.*(\\|/)}}lib"
